### ISSUE TITLE
Feat: Add Toastify Message

### ISSUE THREAD

**Issue by anagha-chaudhari** [Its-Aman-Yadav/Community-Site#791]
### Describe the feature

After sending message in 'Get in Touch' section and after giving feedback under 'Give us your feedback' section, a toastified message will be shown - such as 

Thank you for your feedback,
We'll get in touch soon, or
Please provide all details 

Please assign this issue to me under GSSOC'24

### Add ScreenShots

![site](https://github.com/Its-Aman-Yadav/Community-Site/assets/143149376/e8c4ae63-9d45-4191-9eae-1b87cd06cc4c)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by anagha-chaudhari** [comment]
@Its-Aman-Yadav is it possible to assign this issue to me?

**Comment by Kalivarapubindusree** [comment]
Yeah u can try but someone already raised this issue if u make it more attractive and efficient than other then I will consider urs and add level labels to ur pr is it okay for u @anagha-chaudhari

**Comment by anagha-chaudhari** [comment]
@Kalivarapubindusree Sure no worries, I will try my best! Thank you for assigning

**Comment by Kalivarapubindusree** [comment]
All the very best and happy contribution @anagha-chaudhari
